dashboard/test/controllers/admin_users_controller_test.rb

```diff
@@ -269,5 +269,57 @@ class AdminUsersControllerTest < ActionController::TestCase
     assert_select 'td', text: @facilitator.id.to_s
   end
 
+  test 'bulk_grant_permission grants multiple user_permissions' do
+    sign_in @admin
+    teacher = create(:teacher, username: 'teacher', email: 'teacher@email.xx')
```

Comment:
note you don't need to supply the username or email. FactoryGirl has defaults. This can be 
```ruby
teacher = create :teacher
```

Reply:
Thanks! Those were leftover from me trying to track down my confusion with `create :student`